Lib/test/test_typing.py

```diff
@@ -765,17 +765,17 @@ class C(Generic[*Ts]): pass
             ('tuple[*Ts]',                             '[int, str]',                                     'tuple[int, str]'),
             ('Tuple[*Ts]',                             '[int, str]',                                     'Tuple[int, str]'),
 
-            ('C[*Ts]',                                 '[*tuple_type[int]]',                             'C[*tuple_type[int]]'),  # Should be C[int]
-            ('tuple[*Ts]',                             '[*tuple_type[int]]',                             'tuple[*tuple_type[int]]'),  # Should be tuple[int]
-            ('Tuple[*Ts]',                             '[*tuple_type[int]]',                             'Tuple[*tuple_type[int]]'),  # Should be Tuple[int]
+            ('C[*Ts]',                                 '[*tuple_type[int]]',                             'C[int]'),
```

Comment:
Since these are all consistent now, we can collapse each group of three cases into a single case by replacing `C`/`tuple`/`Tuple` with `generic`.